src/main/scala/com/yotpo/metorikku/metric/stepActions/Sql.scala

```diff
@@ -21,6 +24,7 @@ case class Sql(query: String, dataFrameName: String, showPreviewLines: Int,
     val newDf = sparkSession.sqlContext.sql(query)
     newDf.createOrReplaceTempView(dataFrameName)
     printStep(newDf, dataFrameName)
+    runDQValidation(sparkSession, dataFrameName, dq)
```

Comment:
why passing the spark session? can it be created where u need it? like `SparkSession.getOrCreate()` ?

Reply:
Will It work ok when we use `sparkSession.table(dataFrameName)`?